src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -832,3 +999,56 @@ def __removed_older_complete_status_files(self, status_folder):
 
         self.composite_logger.log_debug("Cleaned up older complete status files: {0}".format(files_removed))
 
+    def __calc_status_size_on_disk(self, status_file_dumps):
+        """ Calculate status file size in bytes on disk """
+        return len(status_file_dumps.encode("utf-8"))
+
+    def __calc_patches_payload_size_on_disk(self, patches):
+        """ Calculate final patches size in bytes (because of escape chars) """
+        first_json_dump = json.dumps(patches)
+        return len(json.dumps(first_json_dump).encode("utf-8"))
+
+    def __size_of_constant_status_data(self, complete_status_file_payload_json, assessment_status_index, installation_status_index):
```

Comment:
Rename complete_status_file_payload_json to status_payload_json

and update function description to:
""" Get the size in bytes of the status payload without patches data  """

Reply:
done